### ISSUE TITLE
Unable to update a form via "Import" when it contains "group" or "repeater"

### ISSUE THREAD

**Issue by benediktblank** [verbb/formie#847]
**Description**
When trying to update a form via the Formie Form Import/Export, forms that contain "group" or "repater" blocks will fail to update.

**Steps to reproduce**
1. Create or import a form with a "group" and a single input field or to get started quicker import this file:
[formie-importTest-3d20c0b0-67df-4472-8533-77b54438a345.zip](https://github.com/verbb/formie/files/8555791/formie-importTest-3d20c0b0-67df-4472-8533-77b54438a345.zip)
It consists of a very simple form:
![image](https://user-images.githubusercontent.com/69143768/165120033-84734545-c777-4db8-9a45-0958fbb7f335.png)
2. Export the created form via the formie exporter or use the file from above.
3. Try to import and update the form
4. Instead of a successful update, the following error appears:
![image](https://user-images.githubusercontent.com/69143768/165118948-10c64c19-2c5d-4492-aacb-8bdd221afff9.png)

While debugging you notice that it fails to validate the fields inside the group.
It seems to think it's not unique. But it being an update an all, it shouldn't matter.
Other fields don't have this issue. Only fields inside "group" and "repeater".

The validation fails somewhere around here:
vendor/yiisoft/yii2/validators/UniqueValidator.php 153
$this->addComboNotUniqueError($model, $attribute);
![image](https://user-images.githubusercontent.com/69143768/165120903-27b70839-bfbf-427c-a051-7c32a7810258.png)

Similar or same issue with "repeater" and the fields inside.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.5.13.2

- Craft version: 3.7.38
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.15",
  "...": "..."
}
```

Then run `composer update`.

**Comment by benediktblank** [comment]
Thank you, that works.

**Comment by engram-design** [comment]
Fixed in [1.5.16](https://github.com/verbb/formie/releases/tag/1.5.16)
